appserver/payara-appserver-modules/microprofile/healthcheck/src/main/java/fish/payara/microprofile/healthcheck/config/MetricsHealthCheckConfiguration.java

```diff
@@ -57,33 +57,36 @@
 public interface MetricsHealthCheckConfiguration extends ConfigBeanProxy, ConfigExtension {
     
     /**
-     * Boolean value determining if the service is enabled or disabled.
-     *
-     * @return
+     * @return a Boolean value determining if the service is enabled or
+     * disabled.
      */
     @Attribute(dataType = Boolean.class, defaultValue = "true")
     String getEnabled();
 
     void setEnabled(String value) throws PropertyVetoException;
 
     /**
-     * String value defines the endpoint of health service.
-     *
-     * @return
+     * @return a String value defines the endpoint of health service.
      */
     @Attribute(defaultValue = "health")
     String getEndpoint();
     
     void setEndpoint(String value) throws PropertyVetoException;
 
     /**
-     * String value defines the attached virtual servers.
-     *
-     * @return
+     * @return a String value defines the attached virtual servers.
      */
     @Attribute(dataType = String.class)
     String getVirtualServers();
 
     void setVirtualServers(String value) throws PropertyVetoException;
     
+    /**
+     * @return a Boolean value determining if the security is enabled or not.
+     */
+    @Attribute(defaultValue = "false", dataType = Boolean.class)
+    String getSecurityEnabled();
+
+    void setSecurityEnabled(String value) throws PropertyVetoException;
```

Comment:
Rename get and set methods to `setAuthorizationRequired` and `getAuthorizationRequired`